cmake/FindRust.cmake

```diff
@@ -231,6 +231,8 @@ if (_RUSTC_VERSION_RAW MATCHES "rustc ([0-9]+)\\.([0-9]+)\\.([0-9]+)(-nightly)?"
     set(Rust_VERSION "${Rust_VERSION_MAJOR}.${Rust_VERSION_MINOR}.${Rust_VERSION_PATCH}")
     if(CMAKE_MATCH_4)
         set(Rust_IS_NIGHTLY 1)
+    else()
+        set(Rust_IS_NIGHTLY 0)
```

Comment:
Will that cause problem for people doing
`if(Rust_IS_NIGHTLY)`  or is 0 falsy?

Reply:
Yes, 0 evaluates to false. From the [documentation](https://cmake.org/cmake/help/latest/command/if.html#basic-expressions)
>  False if the constant is 0, OFF, NO, FALSE, N, IGNORE, NOTFOUND, the empty string, or ends in the suffix -NOTFOUND